spec/models/manageiq/providers/openstack/cloud_manager/refresh_spec_common.rb

```diff
@@ -58,6 +58,7 @@ def assert_with_errors
       expect(OrchestrationStackResource.count).to  eq 0
       expect(OrchestrationStackOutput.count).to    eq 0
       expect(OrchestrationTemplate.count).to       eq 0
+      expect(CloudObjectStoreContainer.count).to   eq storage_data.directories.count
```

Comment:
@hsong-rh Slow down. Will adding these back cause the tests to fail again? I didn't say put them back, I said comment them so we can track the fact that they need to be replaced. Jerry said he'll address the object store failures in his PR if they're commented out here.